plugin.js

```diff
@@ -0,0 +1,50 @@
+const AdminBro = require('admin-bro')
+
+module.exports = {
+  name: 'AdminBroExpressjs',
+  version: '0.1.0',
+
+  register: async (app, options) => {
+    const express = require('express')
```

Comment:
requires should be on top of the file, that developers can see all dependencies in one place.